application/templates/deployment.yaml

```diff
@@ -153,7 +156,10 @@ spec:
         {{- end }}
         {{- if .Values.deployment.env }}
         env:
-{{ toYaml .Values.deployment.env | indent 10 }}
+        {{- range $key, $value := .Values.deployment.env }}
+           - name: {{ include "application.tplvalues.render" ( dict "value" $key "context" $ ) }}
```

Comment:
remove 2 spaces from start. we need to follow one convention. like
env:
- name: abc
  value: xyz